pills/17/fix-function-sum.txt

```diff
@@ -0,0 +1,14 @@
+# Our old friend `sum`
+nix-repl> sum = a: b: a + b
```

Comment:
`sum` usually refers to adding all members of a sequence in functional languages. This function is more commonly called `add`.